spec/integration/app_spec.rb

```diff
@@ -0,0 +1,95 @@
+require "spec_helper"
+require "rack/test"
+require_relative '../../app'
+
+describe Chitter do
+  # This is so we can use rack-test helper methods.
+  include Rack::Test::Methods
+  # We need to declare the `app` value by instantiating the Application
+  # class so our tests work.
+  let(:app) { Chitter.new }
+
+  context "GET to /" do
+    it "returns the home page with peeps on it" do
+      response = get('/')
+      expect(response.status).to eq 200
```

Comment:
One thing Thomas taught me is that you can put this line into: 
after(:each) {  expect(response.status).to eq 200 }

That way you don't have to write it on every test.